hbase-server/src/main/java/org/apache/hadoop/hbase/HBaseTestingUtility.java

```diff
@@ -335,7 +335,7 @@ public HBaseTestingUtility(@Nullable Configuration conf) {
     String dataTestDir = getDataTestDir().toString();
     this.conf.set("fs.defaultFS","file:///");
     this.conf.set(HConstants.HBASE_DIR, "file://" + dataTestDir);
-    LOG.debug("Setting {} to {}", HConstants.HBASE_DIR, dataTestDir);
+    HBaseCommonTestingUtility.LOG.debug("Setting {} to {}", HConstants.HBASE_DIR, dataTestDir);
```

Comment:
Better to create new Logging object instead of using from parent class.

Reply:
Okay